astropy/stats/lombscargle/__init__.py

```diff
@@ -1,8 +1,31 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-"""
-lombscargle
-===========
-AstroPy-compatible implementation of the Lomb-Scargle periodogram.
-"""
-from .core import LombScargle
+# The LombScargle periodogram functionality has been moved to
+# astropy.timeseries.periodograms.bls. The purpose of this file is to provide backward-
+# compatibility during a transition phase. We can't emit a deprecation warning
+# simply on import of this module, since the classes are imported into the
+# top-level astropy.stats, so instead we wrap the main class and emit a
+# warning during initialization.
+
+import warnings
+
+from astropy.timeseries.periodograms.lombscargle import LombScargle as TimeseriesLombScargle
+from astropy.utils.exceptions import AstropyDeprecationWarning
+
+__all__ = ['LombScargle']
+
+
+class LombScargle(TimeseriesLombScargle):
+    """
+    Compute the Lomb-Scargle Periodogram.
+
+    This class has been deprecated and will be removed in a future version.
+    Use `~astropy.timeseries.LombScargle` instead.
```

Comment:
Rendered HTML is confusing because it is not obvious which subpackage it has moved to.
```suggestion
    Use `astropy.timeseries.LombScargle` instead.
```